src/styles/styles.scss

```diff
@@ -0,0 +1,121 @@
+@import 'globals';
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  max-width: 800px;
+  margin: auto;
+  margin-top: 50px;
+  padding-top: 10px;
+  padding-left: 50px;
+  padding-right: 50px;
```

Comment:
Could write this as `padding: 10px 50px 0 50px`

Reply:
done